pkg/kvstore/kvstore_test.go

```diff
@@ -6,31 +6,25 @@ package kvstore
 import (
 	"testing"
 
-	. "github.com/cilium/checkmate"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"github.com/cilium/cilium/pkg/testutils"
 )
 
-func Test(t *testing.T) {
-	TestingT(t)
+func setUpIndependentSuite(tb testing.TB) {
+	testutils.IntegrationTest(tb)
 }
```

Comment:
Nit. I'd drop this function and just call `testutils.IntegrationTest(tb)` directly.